driver/src/main/java/oracle/nosql/driver/ops/TableResult.java

```diff
@@ -377,6 +469,24 @@ public String toString() {
         if (matchETag != null) {
             sb.append("\nmatchETag=").append(matchETag);
         }
+        if (isFrozen) {
+            sb.append("\nisFrozen=").append(isFrozen);
+        }
+        if (isReplicated()) {
+            sb.append("\nisLocalReplicaInitialized=")
+              .append(isLocalReplicaInitialized);
+            sb.append("\nreplicas=[");
+            boolean first = true;
+            for (Replica rep : replicas) {
```

Comment:
Is it ever possible to have isReplicated==true but have a null array of replicas? If so, that would throw an NPE here. Is that checked anywhere else?

Reply:
The [TableResult.isReplicated()](https://github.com/oracle/nosql-java-sdk/blob/feature/mrtable/driver/src/main/java/oracle/nosql/driver/ops/TableResult.java) returns true if replicas array is not null, so it is OK without a check here.

```
    public boolean isReplicated() {
        return replicas != null;
    }
```